tests/TestCase/Utility/HashTest.php

```diff
@@ -1476,6 +1476,17 @@ public function testInsertMulti()
             4 => ['Item' => ['id' => 5, 'title' => 'fifth']],
         ];
         $this->assertEquals($expected, $result);
+
+        $data[3]['testable'] = true;
+        $result = Hash::insert($data, '{n}[testable].Item[id=/\b2|\b4/].test', 2);
```

Comment:
Path syntax already inconsistent for Hash extract and insert methods. I can specify path as "{n}[testable].Item[id=/\b2|\b4/].id" for extract, but not for insert. Don't know anything about other methods, never used them.